code/modules/mob/mob.dm

```diff
@@ -240,6 +240,14 @@
 		return
 
 	face_atom(A)
+	if(!isobserver(src) && !isturf(A))
+		if(A.loc != src)
+			for(var/mob/M in viewers(4, src))
+				if(M == src || M.is_blind())
+					continue
+				if(M.client && M.client.is_preference_enabled(/datum/client_preference/examine_look))
+					to_chat(M, "<span class='tinynotice'><b>\The [src]</b> looks at \the [A].</span>")
```

Comment:
would be cool if this was a list so it pick'd like "glances at" "looks at" "studies" "examines" etc